core/whistle_number_manager-1.0.0/src/wnm_util.erl

```diff
@@ -432,6 +432,10 @@ maybe_convert_to_e164([Regex|Regexs], Converters, Number) ->
             <<Prefix/binary, Root/binary, Suffix/binary>>
     end.
 
+-spec to_e164_without_plus(ne_binary()) -> ne_binary().
+to_e164_without_plus(Number) ->
+    re:replace(to_e164(Number), "[\+]", "", [{'return', 'binary'}]).
```

Comment:
Does this really need to be a regexp?
Looks like pattern matching on the first character, or even a binary comprehension could do the trick here.